examples/src/components/Gallery.js

```diff
@@ -1,6 +1,6 @@
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
-import { css, StyleSheet } from 'aphrodite/no-important';
+import { css, StyleSheet } from '../../../src/aphrodite-no-important';
```

Comment:
I assume the relative path is mandatory for the SSR?

Reply:
To make sure `aphrodite` works on SSR we're passing it through the `/src/aphrodite` and /src/aphrodite-no-important` wrappers.
And this an `example` component of the library, thus the relative path.